docs/siem/detections/rules-ui-create.asciidoc

```diff
@@ -125,58 +107,142 @@ the `delete` and `shadow` arguments, which are used to delete a volume's shadow
-.. _Risk score_: A numerical value between 0 and 100 that correlates with the _Severity_ level. General guidelines are:
+* `Medium`: Alerts that require investigation.
+* `High`: Alerts that require an immediate investigation.
+* `Critical`: Alerts that indicate it is highly likely a security incident has 
+occurred.
+.. _Severity override_ (optional): Select to use source event values to
+override the _Default severity_ in generated alerts. When selected, a UI
+component is displayed where you can map the source event field values to
+severity levels. For example, if you want to map severity levels to `host.name`
+values:
++
+[role="screenshot"]
+image::images/severity-mapping-ui.png[]
+.. _Default risk score_: A numerical value between 0 and 100 that correlates
+with the _Severity_ level. General guidelines are:
 * `0` - `21` represents low severity.
 * `22` - `47` represents medium severity.
 * `48` - `73` represents high severity.
 * `74` - `100` represents critical severity.
-.. For additional options, click *Advanced settings* and fill in any of 
-these fields:
-... _Reference URLs_ (optional): References to information that is relevant to 
-the rule. For example, links to relevant background information.
-... _False positives_ (optional): List of common scenarios that may produce 
-false-positive signals.
-... _MITRE ATT&CK^TM^_ (optional): Relevant MITRE framework tactics and techniques.
-... _Tags_ (optional): Words and phrases used to categorize, filter, and search 
+.. _Risk score override_ (optional): Select to use a source event value to
+override the _Default risk score_ in generated alerts. When selected, a UI
+component is displayed where you can select the source field used for the risk
+score. For example, if you want to use the source event's risk score in
+alerts:
++
+[role="screenshot"]
+image::images/risk-source-field-ui.png[]
+
+. Continue with *one* of the following:
+
+* <<rule-ui-advanced-params>>
+* <<rule-schedule>>
+
+[float]
+[[rule-ui-advanced-params]]
+=== Configure advanced rule settings (optional)
+
+
+. Click *Advanced settings* and fill in these fields:
+.. _Reference URLs_ (optional): References to information that is relevant to 
+the rule. For example, links to background information.
+.. _False positives_ (optional): List of common scenarios that may produce 
+false-positive alerts.
+.. _MITRE ATT&CK^TM^_ (optional): Relevant MITRE framework tactics and techniques.
+.. _Tags_ (optional): Words and phrases used to categorize, filter, and search 
 the rule.
-... _Investigation guide_ (optional): Information for analysts investigating
-signals created by the rule.
+.. _Investigation guide_ (optional): Information for analysts investigating
+alerts created by the rule.
+.. _Author_ (optional): The rule's authors.
+.. _License_ (optional): The rule's license.
+.. _Elastic endpoint exceptions_ (optional): Adds all Elastic Endpoint Security
+rule exceptions to this rule (see <<detections-ui-exceptions>>).
```

Comment:
Throughout the UI/code we've been referencing this as the `Global Endpoint Exception List`, but I like `Elastic Endpoint Security rule exceptions` better, especially for the docs. Will be interesting to see how this terminology expands as we open up more access to modifying this list directly. Let us know if we can improve the UI references as well.